package.json

```diff
@@ -36,7 +36,7 @@
     "fetch-mock": "^8.3.2",
     "jsdoc": "^3.5.5",
     "lie": "^3.3.0",
-    "lit-element": "^2",
+    "lit-element": "^3",
```

Comment:
For some reason the tests keep exploding if I try to bump open-wc/testing to 3. The tests work now so I'm going to say it's OK for now (see many many commits of testing all the combinations 😢)